js/gl.js

```diff
@@ -1269,7 +1269,10 @@ var importObject = {
 
                     FS.loaded_files[file_id] = uInt8Array;
                     wasm_exports.file_loaded(file_id);
-                }
+                } else {
```

Comment:
I am not exactly sure that this will work? Not really sure what to do with this PR, will convert to draft and leave in case someone might find it useful

Reply:
Oh, apparently github do not allow converting to drafts.. than just closing for now

Reply:
Why wouldn't it work? The current behavior is incorrect, do you have another idea for a fix?

Reply:
Does it work? I did not really tried this, if it works fine - sure, lets get it merged